config/prod.exs

```diff
@@ -15,6 +15,9 @@ config :livebook, :iframe_port, 8081
 # Set log level to warning by default to reduce output
 config :logger, level: :warning
 
+# Disable any feature flag in production
+config :livebook, :feature_flags, []
```

Comment:
Unfortunately this doesn't work because the keyword list is merged with the existing value!